### DISCUSSION TITLE
How to add Geo field?

### DISCUSSION THREAD

**Discussion by iyobo** [directus/directus#7567] [Feature Requests]
How does one go about adding a Geo field to directus?

Ideally, this Geo-field would be a Geo-indexed column in the database e.g geopoint in postgis.
The field would have a map and a bunch of fields for address, with the ability to drag-drop to fine-tune longitude latitude.

Would also possibly like to make this field usable by the larger Directus community!

**Comment by rijkvanzanten** [comment]
Heya! As of 18 hours ago, this is natively possible 🙂 You can add a new field of the `geometry` type under Data Model 👍🏻

**Reply by iyobo** [reply]
Woaw! Talk about Providence.
Is this npm-published yet?

**Reply by rijkvanzanten** [reply]
Yuuuup! It's version 9.0.0-rc.91 and up 👍🏻 

Big round of applause to @Oreilles to get that off the ground.

**Reply by iyobo** [reply]
@rijkvanzanten perfect! Then maybe I will divert my energy to improving this field.

Can you point me to the PR that made this possible so I can see how it was done and improve on it?
I plan to add geo-coding to this field.

**Reply by rijkvanzanten** [reply]
@iyobo If you add a Mapbox key to Project Settings, the Map interface / Layout will both show a reverse geo-coding search box 😄 

For everything else map/geo related, I defer to @Oreilles who is _le champion des cartes_

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
